### ISSUE TITLE
[BUG] - tailwind styles aren't working in nextjs standalone build

### ISSUE THREAD

**Issue by Raghava-Ch** [nextui-org/nextui#3415]
### NextUI Version

2.**

### Describe the bug

In simple terms the project just created using `nextui init my_app` built in standalone mode,
bun .next/standalone/server.js - no styles applied.
bun run start - working as expected.

Observations:
Removing `plugins: [nextui()],` from the postcss.config.js file, at least the styles in safe mode is showing up.
With `plugins: [nextui()],` the whole application styles are broken

### Steps to Reproduce the Bug or Issue

1. create new application `nextui init my_app`
2. change output mode to standalone in next.config.js `output: 'standalone'`
3. build the application `bun run build`
4. run the application using `bun .next/standalone/server.js`
5. finally observe the web app in browser, in my case safari

### Expected behavior

tailwind styles should show up in the web page built in standalone mode.

### Operating System Version

macOS Sonoma 14.5 - intel silicon

### Browser

Safari

**Comment by dendenmushy** [comment]
same problem, someone find a way to go ahead of this error?

**Comment by dendenmushy** [comment]
Did you try to remove your .npmrc ? in some case that can fix the problem for Docker user

**Comment by wingkwong** [comment]
Have you setup tailwindcss ?

**Comment by Raghava-Ch** [comment]
> Have you setup tailwindcss ?

Yes, Styles are working as expected when run with `dev` or `start`. Problem with only standalone build.

**Comment by dendenmushy** [comment]
Have you added your .npmrc on the build phase ?

**Comment by Dinista** [comment]
The 'standalone' build doesn't copy the __.next/static__ folder by default. Try copying it to __.next/standalone/.next__ and see if it works.
